### PR TITLE
20.8.0 Release Notes

### PR DESCRIPTION
**Pull request by Itay4**

### CODE REVIEW

File: content-repo/extra-docs/releases/20.8.0.md [hunk 1]
```diff
@@ -0,0 +1,1091 @@
...
+---
+
+### Zimperium Pack v1.0.3
+#### Integrations
+##### Zimperium  
+- Added default classifiers and mappers.
+
+---
+
+### Zoom Feed Pack v1.1.1
```
Comment (ronykoz) [inline content-repo/extra-docs/releases/20.8.0.md:1091]: why there is no info here?

File: content-repo/extra-docs/releases/20.8.0.md [hunk 2]
```diff
@@ -0,0 +1,1091 @@
...
+---
+
+### SplunkPy Pack v1.1.0
+#### Integrations
+##### SplunkPy  
+- Added default classifiers and mappers.
+
+---
+
+### TAXII Feed Pack v1.0.2
```
Comment (ronykoz) [inline content-repo/extra-docs/releases/20.8.0.md:1028]: why there is no info here?

File: content-repo/extra-docs/releases/20.8.0.md [hunk 3]
```diff
@@ -0,0 +1,1091 @@
...
+- Handled dependencies for the playbook.
+
+##### Prisma Cloud Remediation - AWS IAM User Policy Misconfiguration  
+- Handled dependencies for the playbook.
+
+---
+
+### RSA NetWitness Security Analytics Pack v1.0.1
+#### Integrations
+
```
Comment (ronykoz) [inline content-repo/extra-docs/releases/20.8.0.md:920]: need to add info

**Comment by glicht** [comment]
> `- Added default classifiers and mappers.`
> this is only relevant to 6.0 and shouldn't appear here

@yaakovi 6.0 content should still appear as these notes are also for offline 6.0 users. We should just specify `(Requires Server 6.0)`

File: content-repo/extra-docs/releases/20.8.0.md [hunk 4]
```diff
@@ -0,0 +1,1090 @@
...
+##### Sepio Systems  
+
+#### Integrations
+##### Sepio  
+Get Agent, Switches and Events from your Sepio Prime
+
+
+---
+
+### Archer Pack v1.0.0
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.8.0.md:141]: This pack is named `RSA Archer`. Not sure how we got this name. @yaakovi 
```suggestion
### RSA Archer Pack v1.0.0
```

File: content-repo/extra-docs/releases/20.8.0.md [hunk 5]
```diff
@@ -0,0 +1,1090 @@
...
+---
+
+### Base Pack v1.1.7
+#### Scripts
+##### CommonServerPython  
+- Removed an unnecessary variable assignment.
+
+##### SaneDocReports  
+- Updates the sane-doc-reports docker image
+  - Fixes markdown HTML tag inconsistencies.
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.8.0.md:213]: No need for second level bullets. This line and the next 4.
```suggestion
- Fixes markdown HTML tag inconsistencies.
```

File: content-repo/extra-docs/releases/20.8.0.md (hunk 4 above)
Comment (yaakovi) [reply content-repo/extra-docs/releases/20.8.0.md:141]: I saw it missing so I added it manually...

File: content-repo/extra-docs/releases/20.8.0.md [hunk 6]
```diff
@@ -0,0 +1,1090 @@
...
+
+---
+
+### Archer Pack v1.0.0
+#### Integrations
+##### New: RSA Archer v2
+The RSA Archer GRC Platform provides a common foundation for managing policies, controls, risks, assessments and deficiencies across lines of business.
+
+#### Scripts
+##### ArcherCreateIncidentExample
```
Comment (yaakovi) [inline content-repo/extra-docs/releases/20.8.0.md:147]:
```suggestion
##### New: ArcherCreateIncidentExample  
```

**Comment by Itay4** [comment]
@glicht please review

File: content-repo/extra-docs/releases/20.8.0.md [hunk 7]
```diff
@@ -0,0 +1,1090 @@
...
+- Fixed an issue where PDFs with multiple different encoding types were not handled.
+
+##### NumberOfPhishingAttemptPerUser  
+Changed the default query to search only for incidents from the last 30 days.
+
+##### ParseEmailFiles  
+- Fixed an issue where the body text of the email was None.
+- Fixed an issue where the headers of the email were None.
+
+##### If-Then-Else
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.8.0.md:334]:
```suggestion
##### New: If-Then-Else  
```

File: content-repo/extra-docs/releases/20.8.0.md [hunk 8]
```diff
@@ -0,0 +1,1090 @@
...
+
+---
+
+### Logz.io Pack v1.1.0
+#### Integrations
+##### Logz.io  
+Fixed search logs with "human date".
+
+#### Playbooks
+##### Logz.io Indicator Hunting
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.8.0.md:733]:
```suggestion
##### New: Logz.io Indicator Hunting  
```

File: content-repo/extra-docs/releases/20.8.0.md [hunk 9]
```diff
@@ -0,0 +1,1090 @@
...
+Handled dependencies for the playbook.
+
+---
+
+### PCAP Analysis Pack v2.3.0
+#### Playbooks
+##### PCAP File Carving  
+Added new playbook input to validate if detonation is required.
+
+##### PCAP Analysis
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.8.0.md:820]:
```suggestion
##### New: PCAP Analysis  
```

File: content-repo/extra-docs/releases/20.8.0.md [hunk 10]
```diff
@@ -0,0 +1,1090 @@
...
+---
+
+### SafeBreach - Breach and Attack Simulation platform Pack v1.0.4
+#### Integrations
+##### SafeBreach v2  
+Added the *Tags* parameter.
+
+#### Layouts
+#### Playbooks
+##### SafeBreach - Create Incidents per Insight and Associate Indicators
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.8.0.md:960]:
```suggestion
##### New: SafeBreach - Create Incidents per Insight and Associate Indicators  
```

File: content-repo/extra-docs/releases/20.8.0.md [hunk 11]
```diff
@@ -0,0 +1,1090 @@
...
+#### Integrations
+##### SafeBreach v2  
+Added the *Tags* parameter.
+
+#### Layouts
+#### Playbooks
+##### SafeBreach - Create Incidents per Insight and Associate Indicators  
+This is a sub-playbook that creates incidents per SafeBreach insight, enriched with all the related indicators and additional SafeBreach insight contextual information. Used in main SafeBreach playbooks, such as "SafeBreach - Process Behavioral Insights Feed” and "SafeBreach - Process Non-Behavioral Insights Feed".
+
+##### SafeBreach - Process Non-Behavioral Insights Feed
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.8.0.md:963]:
```suggestion
##### New: SafeBreach - Process Non-Behavioral Insights Feed  
```

File: content-repo/extra-docs/releases/20.8.0.md [hunk 12]
```diff
@@ -0,0 +1,1090 @@
...
+#### Playbooks
+##### SafeBreach - Create Incidents per Insight and Associate Indicators  
+This is a sub-playbook that creates incidents per SafeBreach insight, enriched with all the related indicators and additional SafeBreach insight contextual information. Used in main SafeBreach playbooks, such as "SafeBreach - Process Behavioral Insights Feed” and "SafeBreach - Process Non-Behavioral Insights Feed".
+
+##### SafeBreach - Process Non-Behavioral Insights Feed  
+Automatically remediates all non-behavioral indicators
+  generated from SafeBreach Insights. To validate the remediation, it reruns the related
+  insights and classifies the indicators as Remediated or Not Remediated.
+
+##### SafeBreach - Rerun Single Insight
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.8.0.md:968]:
```suggestion
##### New: SafeBreach - Rerun Single Insight  
```

File: content-repo/extra-docs/releases/20.8.0.md [hunk 13]
```diff
@@ -0,0 +1,1090 @@
...
+Automatically remediates all non-behavioral indicators
+  generated from SafeBreach Insights. To validate the remediation, it reruns the related
+  insights and classifies the indicators as Remediated or Not Remediated.
+
+##### SafeBreach - Rerun Single Insight  
+This is an auxiliary sub-playbook that reruns a single insight using
+  a specified Insight Id as an input. It is used to loop over insights as part of
+  the main rerun playbook - "SafeBreach Rerun Insights".
+
+##### SafeBreach - Rerun Insights
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.8.0.md:973]:
```suggestion
##### New: SafeBreach - Rerun Insights  
```

File: content-repo/extra-docs/releases/20.8.0.md [hunk 14]
```diff
@@ -0,0 +1,1090 @@
...
+##### SafeBreach - Rerun Single Insight  
+This is an auxiliary sub-playbook that reruns a single insight using
+  a specified Insight Id as an input. It is used to loop over insights as part of
+  the main rerun playbook - "SafeBreach Rerun Insights".
+
+##### SafeBreach - Rerun Insights  
+Reruns a SafeBreach insight based on Insight ID and waits
+  until it completes.
+
+##### SafeBreach - Compare and Validate Insight Indicators
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.8.0.md:977]:
```suggestion
##### New: SafeBreach - Compare and Validate Insight Indicators  
```

**Comment by glicht** [comment]
@Itay4 something weir with the formatting of ServiceNow:

image

File: content-repo/extra-docs/releases/20.8.0.md [hunk 15]
```diff
@@ -0,0 +1,1090 @@
...
+
+---
+
+### Tufin Pack v1.1.0
+#### Integrations
+##### Tufin  
+Minor updates.
+
+#### Playbooks
+##### Tufin - Enrich IP Address(es)
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.8.0.md:1056]:
```suggestion
##### New: Tufin - Enrich IP Address(es)  
```

File: content-repo/extra-docs/releases/20.8.0.md [hunk 16]
```diff
@@ -0,0 +1,1090 @@
...
+### Tufin Pack v1.1.0
+#### Integrations
+##### Tufin  
+Minor updates.
+
+#### Playbooks
+##### Tufin - Enrich IP Address(es)  
+Added a playbook that enriches a single IP address or multiple IP addresses.
+
+##### Tufin - Get Network Device Info by IP Address
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.8.0.md:1059]:
```suggestion
##### New: Tufin - Get Network Device Info by IP Address  
```

File: content-repo/extra-docs/releases/20.8.0.md [hunk 17]
```diff
@@ -0,0 +1,1090 @@
...
+Minor updates.
+
+#### Playbooks
+##### Tufin - Enrich IP Address(es)  
+Added a playbook that enriches a single IP address or multiple IP addresses.
+
+##### Tufin - Get Network Device Info by IP Address  
+Added a playbook that gets network device information, by IP address.
+
+##### Tufin - Investigate Network Alert
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.8.0.md:1062]:
```suggestion
##### New: Tufin - Investigate Network Alert  
```

File: content-repo/extra-docs/releases/20.8.0.md [hunk 18]
```diff
@@ -0,0 +1,1090 @@
...
+##### Tufin - Enrich IP Address(es)  
+Added a playbook that enriches a single IP address or multiple IP addresses.
+
+##### Tufin - Get Network Device Info by IP Address  
+Added a playbook that gets network device information, by IP address.
+
+##### Tufin - Investigate Network Alert  
+Added a playbook that investigates a network alert.
+
+##### Tufin - Get Application Information from SecureApp
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.8.0.md:1065]:
```suggestion
##### New: Tufin - Get Application Information from SecureApp  
```

File: content-repo/extra-docs/releases/20.8.0.md [hunk 19]
```diff
@@ -0,0 +1,1090 @@
...
+##### Tufin - Get Network Device Info by IP Address  
+Added a playbook that gets network device information, by IP address.
+
+##### Tufin - Investigate Network Alert  
+Added a playbook that investigates a network alert.
+
+##### Tufin - Get Application Information from SecureApp  
+Added a playbook that gets application information from SecureApp.
+
+##### Tufin - Enrich Source & Destination IP Information
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.8.0.md:1068]:
```suggestion
##### New: Tufin - Enrich Source & Destination IP Information  
```

**Comment by Itay4** [comment]
> @Itay4 something weir with the formatting of ServiceNow:
> 
> image

@glicht fixed bullets indentation
image

**Pull request merged**